test_dependabot_file.py

```diff
@@ -54,7 +54,37 @@ def test_build_dependabot_file_with_existing_config_bundler_no_update(self):
         result = build_dependabot_file(repo, False, [], existing_config)
         self.assertEqual(result, expected_result)
 
-    def test_build_dependabot_file_with_existing_config_bundler_with_update(self):
+    def test_build_dependabot_file_with_2_space_indent_existing_config_bundler_with_update(
+        self,
+    ):
+        """Test that the dependabot.yml file is built correctly with bundler"""
+        repo = MagicMock()
+        repo.file_contents.side_effect = lambda f, filename="Gemfile": f == filename
+
+        # expected_result maintains existing ecosystem with custom configuration and adds new ecosystem
+        expected_result = """---
+version: 2
+updates:
+- package-ecosystem: "pip"
```

Comment:
maybe it is just the diff view, but shouldn't there be 2 spaces here on each line?  Same with the rest of the tests.  Default indent is 2, correct?

Reply:
The "- " replaces the last two chars of the indent as shown in these docs. Code for that is here (notice the indent[:-2]) :
```python
dependabot_config = f"""{indent[:-2]}- package-ecosystem: '{ecosystem}'
{indent}directory: '/'
{indent}schedule:
{indent}{indent}interval: 'weekly'
```

That being said, I'm not sure this is the correct thing to do. 😮‍💨 

[The docs](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#example-of-a-basic-setup-for-three-package-managers) demonstrate one spacing scheme:
```yml
version: 2
updates:
  - package-ecosystem: "pip"
    directory: "/"
```

while internally at github (even in the monolith) I've also seen:
```yml
version: 2
updates:
- package-ecosystem: "pip"
  directory: "/"
```

Reply:
I'm inclined to go with the docs on this one and fix our internal "misuse" of the odd alignment.